ui/app/AppLayouts/Chat/views/communities/CommunityNewCollectibleView.qml

```diff
@@ -0,0 +1,260 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Controls.Validators 0.1
+
+import utils 1.0
+
+import "../../../Wallet/controls"
+import shared.panels 1.0
+
+StatusScrollView {
+    id: root
+
+    property var store
+    property int viewWidth: 560 // by design
+
+    signal chooseArtWork
+    signal previewClicked
+
+    // TEMPORAL
+    signal mintCollectible(url artworkSource,
+                           string name,
+                           string symbol,
+                           string description,
+                           int supply,
+                           bool infiniteSupply,
+                           bool transferable,
+                           bool selfDestruct,
+                           int chainId)
+
+    QtObject {
+        id: d
+
+        readonly property bool isFullyFilled: d.artworkSource.toString().length > 0
+                                              && !!nameInput.text
+                                              && !!symbolInput.text
+                                              && !!descriptionInput.text
+                                              && (unlimitedSupplyItem.checked || (!unlimitedSupplyItem.checked && supplyInput.text.length > 0))
+
+
+        readonly property int imageSelectorRectWidth: 280
+
+        property url artworkSource
+        property int networkSelected
+    }
+
+    contentWidth: mainLayout.width
+    contentHeight: mainLayout.height
+    padding: 0
+
+    ColumnLayout {
+        id: mainLayout
+
+        width: root.viewWidth
+        spacing: Style.current.padding
+
+        StatusImageSelector {
+            Layout.preferredHeight: d.imageSelectorRectWidth + 34 /*Label*/
```

Comment:
Why these magic numbers? Should not be needed

Reply:
The component includes a label + the image area (rectangle). In some case (like here), the consumer will want to fix the size of the rectangle itself so we need somehow to provide the label heigh and the plus offset button value to calculate the total size of the component. 
To remove the magic numbers and improve it a bit I've created 2 readonly properties:

```
readonly property int headerHeight: label.implicitHeight + rootImageSelector.spacing
readonly property int buttonsInsideOffset: 10
```

Now the consumer will be able to add these values as part of the total size calculation if they need them! WDYT?